### PR TITLE
ReadMe to add new hpo usecase

### PR DESCRIPTION
**Pull request by kusumachalasani**
ReadMe to detail the steps to add new hpo usecase.

### CODE REVIEW

File: hpo_helpers/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- For each trial in an experiment, following steps are proceeded in sequential order.
+    - Generate the trial config from HPO.
+    - Run the usecase by passing the HPO config. Objective_function result and status is expected from the output of usecase.
+    - Send the objective_function result and status of the usecase back to HPO.
+    - Generate the subsequent trial.
+- After completing the total_trials in an experiment, it generates the recommended HPO config.
+- HPO configuration and metrics from demo usecase for all trials will be updated in experiment-output.csv
+
+
+## To add new usecase
```
Comment (dinogun) [inline hpo_helpers/ReadMe.md:18]: I would suggest update this as
```
Customize the script for your usecase
```

File: hpo_helpers/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            https://github.com/kruize/kruize-demos/blob/fcbac244db5eac5e6254dfb0bd932c629ff399ab/hpo_helpers/runbenchmark.sh#L35
+    - Collect hpo config and benchmark results from all the trials in an experiment into single csv 
+            https://github.com/kruize/kruize-demos/blob/fcbac244db5eac5e6254dfb0bd932c629ff399ab/hpo_helpers/utils.py#L56
+- [getenvoptions.py](getenvoptions.py)
+    - Generate  JDK_JAVA_OPTIONS to be passed to usecase using the JVM + Quarkus tunables from HPO config.
+- [getobjfuncresult.py](getobjfuncresult.py)
+     - Calculate the objective function result based on the results generated by usecase and variables defined in objective function.
+
+##
+**See [HPO API ReadMe](https://github.com/kruize/hpo/blob/main/design/API.md) for more details on HPO API.**
```
Comment (dinogun) [inline hpo_helpers/ReadMe.md:46]: HPO API ReadMe => HPO API

File: hpo_helpers/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Add your usecase to HPO demo script
```
Comment (dinogun) [inline hpo_helpers/ReadMe.md:1]: Please rename the file to README.md to follow convention

File: README.md [hunk 4]
```diff
@@ -16,6 +16,8 @@ Machine learning is a process of teaching a system to make accurate predictions
   Currently it provides a log that consists the results of all the trials and needs to be manually evaluated to see which configuration provided the best result.
 #### pre-req
   It expects minikube to be installed with atleast 8 CPUs and 16384MB Memory.
+#### To add Usecase 
+   Follow [Add HPO UseCase](hpo_helpers/ReadMe.md) to add any usecase to run with Kruize HPO.
```
Comment (dinogun) [inline README.md:20]: Should be
#### Customize it for your usecase
    Follow [Customize the script for your usecase](hpo_helpers/README.md) to customize the demo script for your usecase.

File: hpo_helpers/ReadMe.md (hunk 1 above)
Comment (kusumachalasani) [reply hpo_helpers/ReadMe.md]: Done.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**